build/js/live-editor.output_pjs.js

```diff
@@ -1213,14 +1215,29 @@ window.PJSOutput = Backbone.View.extend({
         this.config = options.config;
         this.output = options.output;
 
-        this.tester = new PJSTester(_.extend(options, {
-            workerFile: "pjs/test-worker.js"
-        }));
-
         this.render();
+        this.build();
+
+        this.globals = {};
+
+        // see if there are any images we should load
+        // right now we're keeping this list in localStorage,
+        // but really it should be injected into output.html
+        // along with the code when output.html is loaded
+        var imageFilenames;
+        if (window.localStorage.imageFilenames) {
+            imageFilenames = JSON.parse(window.localStorage.imageFilenames);
+        } else {
+            imageFilenames = this.getImageFilenames(options.code);
+        }
+        this.cacheImages(imageFilenames, function () {
+            this.injectCode(options.code, function(errors) {
+                console.log(errors);
+            });
+        }.bind(this));
```

Comment:
Start the image caching process before first run.  This could be improved by creating a promise and then waiting for the promise to resolve before attempting to run.